### PR TITLE
create review

### LINKED ISSUE #1: Create Review
- [x] check for product in the `req.body`
- [x] attach user property (set it equal to `req.user.userId`) on to `req.body`
- [x] create review
- [x] don't test yet

### PR DESCRIPTION
**Pull request by khareemah**
Please review create review controller

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -15,12 +16,15 @@ const swaggerDocument = YAML.load('./swagger.yaml');
 const authRouter = require('./routes/authRoute');
 const userRouter = require('./routes/userRoute');
 const productRouter = require('./routes/productRoute');
+const reviewRouter = require('./route/reviewRoute');
```
Comment (fisayo-dmg) [inline app.js]: I think you have a wrong path here, shouldn't it be `'./routes/reviewRoute'`? check the other comment.

File: routes/reviewController.js
Comment (fisayo-dmg) [inline routes/reviewController.js:1]: I think you meant to name this file `reviewRoute.js`

**Review by fisayo-dmg — CHANGES_REQUESTED** [review]
did this code run properly?

File: routes/reviewController.js
Comment (khareemah) [reply routes/reviewController.js:1]: I have tested the route, its working properly now

**Comment by khareemah** [comment]
i have effected the changes

**Review by fisayo-dmg — APPROVED** [review]
nice.

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — APPROVED** [review]

**Review by fisayo-dmg — CHANGES_REQUESTED** [review]
test

**Pull request closed without merging**